src/tests/java/picard/analysis/CollectMultipleMetricsTest.java

```diff
@@ -288,21 +288,41 @@ public void runGcTest(final File input) throws IOException {
                 Assert.assertEquals(metrics.ALIGNED_READS, 600);
                 Assert.assertEquals(metrics.AT_DROPOUT, 7.234062);
                 Assert.assertEquals(metrics.GC_DROPOUT, 4.086217);
-            } else if (metrics.READ_GROUP != null && metrics.READ_GROUP.equals("TestReadGroup1")) { //Library 1
+                Assert.assertEquals(metrics.GC_NC_0_20, 0.0);
```

Comment:
where did the numbers for these tests come from? could you write a comment about how they were found?